vertical-pod-autoscaler/e2e/v1/actuation.go

```diff
@@ -41,13 +41,15 @@ import (
 	framework_rs "k8s.io/kubernetes/test/e2e/framework/replicaset"
 	framework_ss "k8s.io/kubernetes/test/e2e/framework/statefulset"
 	testutils "k8s.io/kubernetes/test/utils"
+	admissionapi "k8s.io/pod-security-admission/api"
```

Comment:
Would it make sense to use a different package shorthand, to avoid confusion with admission.k8s.io API? How about podsecurity ?

Reply:
Done.